### PR TITLE
Update azure_rm_hdinsightcluster related document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_hdinsightcluster related document。

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_hdinsightcluster/azure_rm_hdinsightcluster_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_hdinsightcluster.py [hunk 1]
```diff
@@ -85,25 +85,25 @@
...
+                            - SSH password.
     storage_accounts:
         description:
             - The list of storage accounts in the cluster.
         type: list
         suboptions:
             name:
                 description:
-                    - Blob storage endpoint.
+                    - Blob storage endpoint. eg. storage_account_name.blob.core.windwos.net.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_hdinsightcluster.py]:
```suggestion
                    - Blob storage endpoint. For example: storage_account_name.blob.core.windows.net.
```

**Review by acozine — COMMENTED** [review]
Generally looks good - two small fixes. We try to avoid Latin ("e.g.") in our documentation. Also, there was a typo in the example.

File: lib/ansible/modules/cloud/azure/azure_rm_hdinsightcluster.py (hunk 1 above)
Comment (Fred-sun) [reply lib/ansible/modules/cloud/azure/azure_rm_hdinsightcluster.py]: Thanks for your review, I will update!

**Comment by Fred-sun** [comment]
ready_for_review

**Comment by acozine** [comment]
Thanks @Fred-sun!

**Pull request merged**